fineract-doc/build.gradle

```diff
@@ -40,7 +40,7 @@ asciidoctorj {
         pdf.version '1.6.2'
         diagram.version '2.2.1'
         epub.version '1.5.1'
-        revealjs.version '4.1.0'
```

Comment:
By removing this dependency the ascii doc generation is still working?

Reply:
Yes, This is causing the missed library `okhttp-digest` that is not more available. Even the new version (4.0.0) for asciidoctor are still using the same dependency.

Reply:
My question was whether ascii doc generation is still working even if this dependency got removed?